server/graphql/v1/mutations/collectives.js

```diff
@@ -514,8 +514,9 @@ export async function archiveCollective(_, args, req) {
   }
 
   // TODO: cascade deactivation to EVENTs and PROJECTs?
+  await collective.changeHost(null);
```

Comment:
`changeHost` is taking care of the recursive with events & projects, which is a really nice addition. An interesting follow-up could be to set `deactivatedAt` on all children and get rid of this TODO.

Reply:
Followup in https://github.com/opencollective/opencollective-api/pull/8027